test/utils/assertions.ts

```diff
@@ -0,0 +1,59 @@
+import { ChannelIdentifier } from 'lumino-js-sdk';
+import { sleep } from '.';
+import { LuminoNode } from '../../src/types/node';
+import { ChannelState } from '../common';
+
+export function given(node: LuminoNode): ExpectationContext {
+    return new ExpectationContext(node);
+}
+
+export class ExpectationContext {
+    constructor(private node: LuminoNode) {}
+
+    expectChannel(identifier: ChannelIdentifier): ChannelExpectation {
+        return new ChannelExpectation(this.node, identifier);
+    }
+}
+
+export class ChannelExpectation {
+    constructor(
+        private node: LuminoNode,
+        private identifier: ChannelIdentifier
+    ) {}
+
+    async toHaveBalance(expected: number, timeout = 1000, retries = 5) {
+        let actual = await this.node.client.sdk.getChannel(this.identifier);
+        for (let i = 0; expected !== actual.balance && i < retries; i++) {
+            await sleep(timeout);
+            actual = await this.node.client.sdk.getChannel(this.identifier);
+        }
+        expect(actual.balance).toEqual(expected);
+    }
+
+    async toHaveDeposit(expected: number, timeout = 1000, retries = 5) {
+        let actual = await this.node.client.sdk.getChannel(this.identifier);
+        for (let i = 0; expected !== actual.total_deposit && i < retries; i++) {
+            await sleep(timeout);
+            actual = await this.node.client.sdk.getChannel(this.identifier);
+        }
+        expect(actual.total_deposit).toEqual(expected);
+    }
+
+    async toBeInState(expected: string, timeout = 1000, retries = 5) {
+        let actual = await this.node.client.sdk.getChannel(this.identifier);
+        for (let i = 0; expected !== actual.state && i < retries; i++) {
+            await sleep(timeout);
+            actual = await this.node.client.sdk.getChannel(this.identifier);
+        }
+        expect(actual.state).toEqual(expected);
+    }
+
+    async toBe(expected: ChannelState) {
```

Comment:
I'm ok with this function, i think is just to verify an expected state, if is not used let's keep it since we could use it later in other scenarios

Reply:
@JONAF2103 

what is your reaction when you are inspecting a codebase which has a function which isn't used anywhere? do you ever think "great to have this in case i need it layer"?

we have git, we can add it back if and when we need to. 

unused code just lying around is just confusing, and bloats the project unnecessarily

Reply:
In general i agree with your statement, but since this is a framework for testing, is better for new people to start working on this and have those functions already available when they are writing a new test than assuming those doesn't exists. Also if you start removing unused code only because we don't use it right now but ignoring the potential that could have in some future cases then i don't know what to say, think about this project is far to be completed, actually we are in development phase all time since we always add new tests, if we have some new idea that could be used i think is ok to keep it, unless of course we know that the code will never be used.